README.md

```diff
@@ -1 +1,66 @@
-# books-modules
\ No newline at end of file
+# AWESOME BOOKS PROJECT
+A collection of books
+
+## DESCRIPTION
+In this project, a basic website that allows users to add/remove books from a list was deployed using JavaScript objects and arrays.
+The DOM is dynamically modified according to the addings and removings operations.
+The localSorage is updated also.
+
+## BUILT WITH
+- *Visual Studio Code*
+
+- *Github Desktop application*
+
+- *Linters for HTML/CSS/JS*
+
+## HISTORY OF CHANGES
+**2022 May 23**: First version released and presented for review.
+
+**2022 May 24**: Project modified to include a class with methods for adding and removing books. CSS and HTML files were updated to style the page.
+
+**2022 May 25**: Project modified to include three separated sections: list books, add books and contact using SPA methodology.
+
+## LIVE DEMO: 
+
+[Awesome Books Project](https://hanszizold.github.io/Awesome_Books/)
+
+## Getting Started
+
+**To get a local copy up and running follow these simple example steps**.
+
+### Prerequisites
+
+- You need to have a code editor to be able to run the code. VSCode is most preferred.
+
+### Setup
+
+- To clone the code press in the green button at the top right corner in this page
+- Select the method that you want to clone
+
+### Install
+
+- Run it in your code editor
+
+### Usage
+
+- You can download an extension for VSCode named "Live Server" so you can see the actual project working in the browser
+
+### Authors
+
+Hans Paul Zizold
+    
+  [GITHUB](https://github.com/HansZizold)
+
+  [LINKEDIN](https://www.linkedin.com/in/hans-paul-zizold-37129037/)
+
+Shay Craig
+    
+  [GITHUB](https://github.com/craigs40)
+
+Andres Garzon Maldonado
+    
+  [GITHUB](https://github.com/andgarzonmal)
+  [LINKEDIN](https://www.linkedin.com/in/andres-garzon-maldonado-951a2a180/)
+
+
+## ACKNOWLEDMENTS: 
```

Comment:
- [OPTIONAL] Please it will be nice to add content to your acknowledgement section, as the will make your readme file look more professional.